src/pymor/models/iosys.py

```diff
@@ -29,6 +31,21 @@ def sparse_min_size(value=1000):
     return value
 
 
+_preset_assertions = FrozenDict(
+    {
+        "c_lrcf": lambda A, val: isinstance(val, VectorArray) and val in A.source,
+        "o_lrcf": lambda A, val: isinstance(val, VectorArray) and val in A.range,
+        "c_dense": lambda A, val: isinstance(val, np.ndarray)
+        and val.shape == (A.source.dim, A.range.dim),
+        "o_dense": lambda A, val: isinstance(val, np.ndarray)
+        and val.shape == (A.range.dim, A.source.dim),
+        "hsv": lambda A, val: isinstance(val, np.ndarray) and val.ndim == 1 and len(val) == A.source.dim,
+        "h2_norm": lambda A, val: isinstance(val, float) and val >= 0,
```

Comment:
This will not work if `val` is an `int` or a NumPy scalar.

```suggestion
        "h2_norm": lambda A, val: val >= 0,
```